tests/tracer/test_tracer.py

```diff
@@ -1654,3 +1654,29 @@ def test_fork_manual_span_different_contexts(tracer):
     _, status = os.waitpid(pid, 0)
     exit_code = os.WEXITSTATUS(status)
     assert exit_code == 12
+
+
+def test_fork_different_contexts_span_tags(tracer):
```

Comment:
```suggestion
def test_fork_pid(tracer):
```
Just to make the test name a little clearer for what we're testing for 😄